bfdd/bfdd_vty.c

```diff
@@ -413,13 +428,15 @@ static void _display_peer_counter_json_iter(struct hash_bucket *hb, void *arg)
 	json_object_array_add(jo, jon);
 }
 
-static void _display_peers_counter(struct vty *vty, char *vrfname, bool use_json)
+static void _display_peers_counter(struct vty *vty, char *vrfname,
+				   bool use_json)
 {
 	struct json_object *jo;
 	struct bfd_vrf_tuple bvt;
 
-	memset(&bvt, 0, sizeof(struct bfd_vrf_tuple));
+	memset(&bvt, 0, sizeof(bvt));
```

Comment:
since struct bfd_vrf_tuple is very simple. struct bfd_vrf_tuple bvt = {0}; would be much cheaper than memset.

Reply:
Done